pkgs/development/python-modules/mocket/default.nix

```diff
@@ -78,15 +79,19 @@ buildPythonPackage rec {
 
   _darwinAllowLocalNetworking = true;
```

Comment:
I think this should be `__darwinAllowLocalNetworking`, but it not works too...

Reply:
added another commit to correct this -- agree likely not related though